Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Big artifact pictures for `A Bigger Interface` mod compatibility

## Changelog:
- Added big artifact pictures for `A Bigger Interface` mod compatibility



## How to test:

- Check that I didn't break small pictures.
- `get_all_artifacts` console command can help you check jewelry and non-elemental weapon icons.
